tasks/ui_framework_test.js

```diff
@@ -0,0 +1,36 @@
+const jest = require(`jest`);
```

Comment:
Might have been looking at Cloud UI code (as that code has backticks in all the places). Maybe there's an eslint we can use to error on this? So easy to forget.